components/admin_console/admin_definition.jsx

```diff
@@ -3099,6 +3114,18 @@ const AdminDefinition = {
                             it.stateIsFalse('SamlSettings.Enable'),
                         ),
                     },
+                    {
+                        type: Constants.SettingsTypes.TYPE_TEXT,
+                        key: 'SamlSettings.AdminAttribute',
+                        label: t('admin.saml.adminAttrTitle'),
+                        label_default: 'Admin Attribute:',
+                        placeholder: t('admin.saml.adminAttrEx'),
+                        placeholder_default: 'E.g.: "usertype=Admin" or "isAdmin=true"',
+                        help_text: t('admin.saml.adminAttrDesc'),
+                        help_text_default: '(Optional) The attribute in the SAML Assertion that will be used to apply a admin role to users in Mattermost.\n \nNote: If this attribute is removed/changed from your admin user in SAML and the user is still active, they will not be demoted to a member and will retain their Admin role. Admins can be demoted in **System Console > User Management**.\n \n \nExisting members that are identified by this attribute as a admin will be demoted from a admin to a member when they are asked to login next. The next login is based upon Session lengths set in **System Console > Session Lengths**. It is highly recommend to manually demote users to members in **System Console > User Management ** to ensure access is restricted immediately.',
```

Comment:
@thefactremains Help with this one as well. The user will retain Admin role until they next login.

Reply:
> Admin Filter:
> (Optional) Enter an AD/LDAP filter to use for designating System Admins. The users selected by the query will have access to your Mattermost server as System Admins. By default, System Admins have complete access to the Mattermost System Console. 
> 
> Existing members that are identified by this attribute will be promoted from member to System Admin upon next login. The next login is based upon Session lengths set in System Console > Session Lengths. It is highly recommend to manually demote users to members in System Console > User Management to ensure access is restricted immediately.
> 
> Note: If this filter is removed/changed, System Admins that were promoted via this filter will be demoted to  members and will not retain access to the System Console. When this filter is not in use, System Admins can be manually promoted/demoted in System Console > User Management.

Reply:
@justinegeffen The above comment describes the functionality of this filter. Not sure where you want to document it. Let's discuss on our weekly?